proxy/secret.go

```diff
@@ -0,0 +1,66 @@
+package proxy
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/openfaas/faas-cli/schema"
+)
+
+//GetSecretList get secrets list
+func GetSecretList(gateway string, tlsInsecure bool) ([]schema.Secret, error) {
+	var results []schema.Secret
+
+	if tlsInsecure {
```

Comment:
I think we only show this if tlsInsecure is off? @LucasRoesler?

Reply:
yeah, I think the check should be inverted here.  Relatedly, the  doc string on this function should document the args to make it clearer how `tlsInsecure` should behave